sdcm/results_analyze/__init__.py

```diff
@@ -100,21 +100,17 @@ def _test_version(self, test_doc):
         self.log.error('Scylla version is not found for test %s', test_doc['_id'])
         return None
 
-    def get_events(self):
+    def get_events(self, event_severity=None):
```

Comment:
@fgelcer build a new function that would be able to take the the specific events you want out of those lists
I don't agree we should use the severity like that.

Reply:
there is a function that will get specific event, giving the required severity to this very same function

Reply:
We discussed it in a review meeting and we agreed to use DEBUG severity for stalls.